ha/core/config/config_manager.py

```diff
@@ -126,3 +127,19 @@ def get_hw_env() -> str:
         Get if system is running on VM or actual h/w.
         """
         return Conf.get(const.HA_GLOBAL_INDEX, f"CLUSTER_MANAGER{_DELIM}env")
+
+    @staticmethod
+    def get_node_name(node_id: str) -> str:
```

Comment:
We already have a _get_node_name   function in node_stop  branch in pcs_controller.py file.

Let us have such functions in same places, with same names; across branches;
 so that when everything gets merged to main, there are no duplicates
 @indrajitzagade  @ajay-paratmandali

Reply:
@ArchanaLimaye Already gave comment in https://github.com/Seagate/cortx-ha/pull/544/ regarding this.

Reply:
yes i observed same 

in node stop it is private function and also getting node name will be part of config

We can not put it in cluster manager
can we add todo in node stop

https://github.com/Seagate/cortx-ha/pull/544
can we handle in above pr

@akash2144

Reply:
oh ok
https://github.com/Seagate/cortx-ha/pull/544/files

In PR it is already taken care 
moved to configmanager

Reply:
okay